server/src/main/java/org/elasticsearch/index/mapper/MetadataFieldMapper.java

```diff
@@ -129,7 +129,7 @@ boolean isConfigured() {
         }
 
         @Override
-        public abstract MetadataFieldMapper build(BuilderContext context);
+        public abstract MetadataFieldMapper build(ContentPath contentPath);
```

Comment:
This is actually unneeded for MetadataFieldMappers (they are always built at the root level) so we can remove this in a follow-up